clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -594,6 +594,15 @@ public static <T> String join(T[] strs, String separator) {
         return join(Arrays.asList(strs), separator);
     }
 
+    /**
+     * Create a string representation of a collection joined by ", ".
+     * @param collection The list of items
+     * @return The string representation.
+     */
+    public static <T> String join(Collection<T> collection) {
```

Comment:
Do we really need this method? Why can't we call the existing `join()` method and pass the separator? If you want to keep it, since it's only used in tools let's move it to the tools module.

Reply:
Method removed